src/components/progress-circle/progress_circle.ts

```diff
@@ -0,0 +1,102 @@
+import {Component, ViewEncapsulation, ElementRef, Attribute} from 'angular2/core';
+import {isPresent, CONST} from 'angular2/src/facade/lang';
+
+
+// TODO(josephperrott): Benchpress tests.
+
+/** Display modes of Progress Circle */
+@CONST()
+class ProgressMode {
+  @CONST() static DETERMINATE = 'determinate';
+  @CONST() static INDETERMINATE = 'indeterminate';
+}
+
+
+/**
+ * <md-progress-circle> component.
+ */
+@Component({
+  selector: 'md-progress-circle',
+  inputs: ['value'],
+  host: {
+    '[attr.mode]': 'mode',
+    'role': 'progressbar',
+    'aria-valuemin': '0',
+    'aria-valuemax': '100',
+    '[attr.aria-valuenow]': 'value'
```

Comment:
Can you change the `attr.mode` and `attr.aria-valuenow` to use a `HostBinding` decorator on the properties?
(this API was added after I wrote the original progress-linear in angular/angular)